eclass/flag-o-matic.eclass

```diff
@@ -438,7 +470,23 @@ strip-flags() {
 	return 0
 }
 
+# @FUNCTION: test-flag-PROG
+# @USAGE: <compiler> <flag>
+# @INTERNAL
+# @DESCRIPTION:
+# Returns shell true if <flag> is supported by given <compiler>, else returns shell false.
 test-flag-PROG() {
+	[[ ${EAPI} == [0-7] ]] ||
+		die "Internal function ${FUNCNAME} is not available in >=EAPI-8."
```

Comment:
Is anybody actually using these internals?

Reply:
Not in Gentoo ebuild repo, but better be safe than sorry.

Reply:
I was about to write about how I was unsure about providing this safety net, and then I realised it was never prefixed with a '_' to begin with. As you were!